modules/dailymotionBidAdapter.js

```diff
@@ -90,6 +92,22 @@ function getVideoMetadata(bidRequest, bidderRequest) {
   return videoMetadata;
 }
 
+/**
+ * Check if user sync is enabled for Dailymotion
+ *
+ * @return boolean True if user sync is enabled
+ */
+function isUserSyncEnabled() {
+  const syncEnabled = deepAccess(config.getConfig('userSync'), 'syncEnabled');
+
+  if (!syncEnabled) return false;
+
+  const canSyncWithIframe = syncEnabled && userSync.canBidderRegisterSync('iframe', 'dailymotion');
```

Comment:
Now you added `if (!syncEnabled) return false;` you can delete `syncEnabled &&` in `canSync...` iframe and pixel because it's already tested